synth.py

```diff
@@ -34,3 +27,4 @@
   ".eslintignore",
   ".prettierignore"
 ])
+subprocess.run(['npm', 'run', 'generate'])
```

Comment:
that's nice and simple.

Reply:
Couldn't do this before because generation was taking way too long and timing out.  The `gts`, download, and sample gen PRs worked us towards this